tests/basic/kubevirt_test.go

```diff
@@ -25,15 +27,21 @@ var _ = Describe("{AddNewDiskToKubevirtVM}", func() {
 		stepLog := "schedule a kubevirtVM"
 		Step(stepLog, func() {
 			for i := 0; i < Inst().GlobalScaleFactor; i++ {
-				contexts = append(contexts, ScheduleApplications("test")...)
+				appCtxs = append(appCtxs, ScheduleApplications("test")...)
 			}
 		})
-		defer DestroyApps(contexts, nil)
-		ValidateApplications(contexts)
+		defer DestroyApps(appCtxs, nil)
+		ValidateApplications(appCtxs)
+		for _, appCtx := range appCtxs {
+			bindMount, err := kubevirt.IsVMBindMounted(appCtx, Inst().S, Inst().V, true)
+			log.FailOnError(err, "Failed to verify bind mount")
+			dash.VerifyFatal(bindMount, true, "Failed to verify bind mount")
+		}
+
 		stepLog = "Add one disk to the kubevirt VM"
 		Step(stepLog, func() {
 			log.InfoD(stepLog)
-			success, err := kubevirt.AddDisksToKubevirtVM(contexts, numberOfVolumes, "0.5Gi", Inst().N)
+			success, err := kubevirt.AddDisksToKubevirtVM(appCtxs, numberOfVolumes, "0.5Gi", Inst().N)
```

Comment:
test should verify that the new disk is bind-mounted as well

Reply:
Added check for this as well.